pkg/kubecost/allocation.go

```diff
@@ -934,11 +972,38 @@ func (a *Allocation) add(that *Allocation) {
 	a.NetworkCostAdjustment += that.NetworkCostAdjustment
 	a.LoadBalancerCostAdjustment += that.LoadBalancerCostAdjustment
 
+	// Sum LoadBalancer Allocations
+	a.LoadBalancers = a.LoadBalancers.Add(that.LoadBalancers)
```

Comment:
We probably wouldn't expect this to be _too_ expensive, but in environment with lots of load balancers, and lots of relationships between pods and load balancers, I wonder how heavy this gets. Allocating memory for maps within loops (i.e. the `Clone()` on line 985 happens for each and every allocation addition, and then allocating for map resizing on insert) is something to be careful with.

I bring it up here because we're no longer behind a flag, it seems. This happens on the main path for `Allocation.Add`. That said... I'm not terribly afraid of degraded performance here because this API is already slow, and not used by the UI anywhere. Just something to be mindful of.